test/options.test.ts

```diff
@@ -36,11 +39,15 @@ import {
 
 describe('options', () => {
   it('verify default options', () => {
+    const getInstrumentationsStub = sinon
+      .stub(instrumentations, 'getInstrumentations')
+      .returns([]);
```

Comment:
Is this needed because now in tests it defaults to a list containing http instrumentation? Can we add a comment here to explain it?

Reply:
Yeah, http instrumentation is part of dev dependencies to load the types, which makes `getInstrumentations` not be empty. Adding a comment